tests/ui/feature-gates/issue-43106-gating-of-builtin-attrs-error.stderr

```diff
@@ -127,8 +127,17 @@ LL | #![inline]
 error: `macro_export` attribute cannot be used at crate level
   --> $DIR/issue-43106-gating-of-builtin-attrs-error.rs:12:1
    |
-LL | #![macro_export]
-   | ^^^^^^^^^^^^^^^^
+LL |   #![macro_export]
+   |   ^^^^^^^^^^^^^^^^
+...
+LL | / mod inline {
+LL | |
+LL | |
+LL | |
+...  |
+LL | |
+LL | | }
+   | |_- the inner attribute doesn't annotate this module
```

Comment:
IIRC, we had a "head span" in addition to the whole item span, I think it could be used here.

Reply:
Used `ident.span` instead, which should work well enough for this purpose.